src/components/screens/OpeningTimesCard.js

```diff
@@ -32,15 +32,15 @@ export const OpeningTimesCard = ({ openingHours }) => (
 
         return weekday ? (
           <View key={index} style={styles.divider}>
-            <View>
-              <BoldText style={{ marginBottom: normalize(3) }}>{weekday}</BoldText>
-            </View>
-            <WrapperRow style={{ justifyContent: 'space-between' }}>
+            <BoldText style={styles.marginBottom}>{weekday}</BoldText>
+
+            <WrapperRow space>
               {(!!timeFrom || !!timeTo) && (
                 <TimeBox>
                   {!!timeFrom && (
                     <RegularText>{momentFormat(timeFrom, returnFormat)} -</RegularText>
                   )}
+                  {(!!timeFrom || !!timeTo) && <RegularText> -</RegularText>}
```

Comment:
now you have the ` -` from the line before and additionally a ` -` through this new line?